src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -442,4 +445,33 @@ private function checkIfEntryAlreadyExists(Entry $entry)
     {
         return $this->get('wallabag_core.entry_repository')->findByUrlAndUserId($entry->getUrl(), $this->getUser()->getId());
     }
+
+    /**
+     * Share entry content.
+     *
+     * @param Entry $entry
+     *
+     * @Route("/share/{uuid}", requirements={"uuid" = ".+"}, name="share")
+     * @Cache(maxage="25200", public=true)
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function shareEntryAction(Entry $entry)
+    {
+        return $this->render(
```

Comment:
What if someone doesn't allow public url and a user _magically_ found an uuid entry, there are no check that'll disallow viewing the content.
This'll require a test too.